src/features/filter.ts

```diff
@@ -1,10 +1,58 @@
-export const actions = { /* put action creators here */};
+import { ActionTypes } from '../types/Actions';
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+type StateType = {
+  searchedTitle: string;
+  selectedStatus: Status;
```

Comment:
```suggestion
type StateType = {
  query: string;
  status: Status;
```
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/96409563/63a6e813-4d50-4869-836f-cb3ff2e99318)